Essentials/src/com/earth2me/essentials/User.java

```diff
@@ -801,6 +804,7 @@ public void setVanished(final boolean set) {
                 p.showPlayer(getBase());
             }
             setHidden(false);
+            this.getBase().removeMetadata("vanished", (Plugin)this.ess);
```

Comment:
Cool kids just set it false! :D

Reply:
Eh, but if that were the case then people would expect the tag to always exist. I'm against mythical tri-state booleans.